rs3-full-boss-guides/magister.txt

```diff
@@ -1,178 +1,310 @@
-      },
-      {
-        "name": "__Unstable Mixture__",
-        "value": "⬥ A bomb is thrown in the general vicinity of the player. [Example](https://imgur.com/EqOep9I.mp4)\n⬥ It explodes after the bar above it fills, dealing typeless damage in a 5x5 area to whatever is near it.\n\u00a0\u00a0\u00a0\u00a0• Deals 4000 damage to the player and the Magister and 2000 damage to the player's familiar.\n\u00a0\u00a0\u00a0\u00a0• Can be clicked to kick away.\n\u200B"
-      },
-      {
-        "name": "__Asphyxiate__",
-        "value": "⬥ The Magister will asphyxiate the player, dealing ~1500-3000 damage for 3 hits and healing off the base damage. [Example](https://imgur.com/WTSVZKC.mp4)\n\u00a0\u00a0\u00a0\u00a0• This ability does not stun and can be dealt with by moving or by using a melee threshold.\n\u00a0\u00a0\u00a0\u00a0• The Magister will continue auto attacking mid asphyxiate if stunned.\n\u200B"
-      },
-      {
-        "name": "__Restore Me__",
-        "value": "⬥ The Magister will regain several thousand lifepoints. [Example](https://imgur.com/3siuSRJ.mp4)\n⬥ More likely to use this after falling below 50k lifepoints and it is often the first special attack he will use at that point.\n\u00a0\u00a0\u00a0\u00a0• Can be avoided by stun-locking him, dealing damage and killing him before he can spec.\n\u200B"
-      },
-      {
-        "name": "__Minions__",
-        "value": "⬥ Only happens once the Magister has performed a sufficient number of special attacks beforehand. [Example](https://imgur.com/4rbODea.mp4)\n\u00a0\u00a0\u00a0\u00a0• Two Soul Devourer minions spawn using random combat styles.\n\u00a0\u00a0\u00a0\u00a0• They have 10,000 HP and remain until either they die or the fight ends.\n\u00a0\u00a0\u00a0\u00a0• It is recommended to kill them unless the fight is at least halfway over.\n\u200B"
-      }
-    ]
-  }
-}
-.embed:json
+The Magister can use a random special attack, every 3 auto attacks. Here we cover what specials he has access to and how to deal with them.
+
+**Note that most players will tend to ignore most mechanics and just simply DPS through them with <:soulsplit:615613924506599497> to heal back any lost health.**
+
 .
-> **__Protecting Your Familiar__**
-.tag:familiars
-The main recommended familiars are a Ripper Demon <:ripperpouch:703581275453128714>, a Nihil <:shadownihil:854475441946230855> <:bloodnihil:854475442169053194> <:smokenihil:854475442227642388>, or a Blood Reaver <:reaverpouch:839903693837959228> (high end melee only). All of these familiars have relatively low lifepoints, particularly the ancient summoning familiars, so some attention is needed in order to keep your familiar alive.
+__**Power Blast**__
+.img:https://i.imgur.com/S1fJmLr.gif
+⬥ **Magister hits his staff on the ground, doing a red lightning attack for high ranged damage (~2.5k).**
```

Comment:
"The Magister hits the ground with his staff, unleashing a red lightning attack for high ranged damage (~2.5k)."